source/automation.txt

```diff
@@ -6,48 +6,120 @@ Automation
+maintenance of your {+service+} {+clusters+} and streamline operations,
+reduce manual interventions, and ensure consistent, policy-compliant
+deployments. These tools offer you the flexibility to work with
+|service| through direct scripting, a Kubernetes native workflow, or
+using Infrastructure as Code (IaC) resources.    
 
 {+service+} Features and Best Practices for Automation
 ------------------------------------------------------
 
-Content here
-
-Examples
---------
-
-The following examples <perform this action> using |service|
-:ref:`tools for automation <arch-center-automation>`.
-
-These examples also apply other recommended configurations, including:
-
-.. tabs::
-
-   .. tab:: Dev and Test Environments
-      :tabid: devtest
-
-      .. include:: /includes/shared-settings-clusters-devtest.rst
-
-   .. tab:: Staging and Prod Environments
-      :tabid: stagingprod
-
-      .. include:: /includes/shared-settings-clusters-stagingprod.rst
-
-.. tabs::
-
-   .. tab:: CLI
-      :tabid: cli
-
-      Content here
-
-   .. tab:: Terraform
-      :tabid: Terraform
-
-      Content here
+You can automate the management, provisioning, and configuration of
+|service| building blocks like database users and roles, |service|
+{+clusters+}, projects and organizations. Additionally, you can automate 
+setting up and managing of additional {+clusters+} resources, including
+enabling auto-scaling and disk scaling, creating and updating
+multi-cloud {+clusters+}, monitoring {+cluster+} performance and health,
+automating backups and restores, defining backup policies, and many more.   
+
+You can align your choice of tools with your preferred workflow to
+ensure seamless integration of |service-fullname| into your existing
+processes. The following |service-fullname| tools allow you to easily
+deploy and manage |service| at scale with repeatable, accurate, and
+scalable processes. 
+
+{+atlas-admin-api+} 
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The {+atlas-admin-api+} provides a |rest|\ful interface that allows you
+to leverage your preferred client such as cURL, Postman, and so on to
+directly interact with |api| endpoints that correspond to |service|
+resources.   
+
+To learn more, see :ref:`atlas-admin-api-access`.
+
+GoSDK Client 
+~~~~~~~~~~~~
+
+The GoSDK client simplifies interaction with the {+atlas-admin-api+} by
+providing pre-built functions and full |api| endpoint coverage. The SDK
+provides platform-specific and GoLang language-specific tools,
+libraries, and documentation to help build applications quickly and
+easily.  
+
+To learn more, see :ref:`Atlas Go SDK <atlas-admin-sdk>`.
+
+Infrastructure as Code (IaC) Integrations 
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The integrations with the following :abbr:`IaC (Infrastructure as Code)`
+tools automate the deployment of |service| resources.
+
+.. list-table:: 
+   :stub-columns: 1 
+
+   * - HashiCorp Terraform MongoDB Atlas Provider
```

Comment:
Can we take the IaC options out of the table and just use subheadings? I think it would make it easier to notice them as equivalent options to the ones above